core/src/main/scala/org/apache/spark/status/storeTypes.scala

```diff
@@ -399,6 +399,21 @@ private[spark] class ExecutorStageSummaryWrapper(
 
 }
 
+private[spark] class SpeculationStageSummaryWrapper(
```

Comment:
ditto.

Reply:
how about this one does this one need to be removed? no I guess?